### PR TITLE
Do refactor for Ansible jobs to keep struct consistent

### PR DESCRIPTION
**Pull request by kiwik**
- Add specific name attribue for all of task
- Replace OS_TENANT_* to OS_PROJECT_*
- Use consist way to get API_HOST_IP

Closes theopenlab/openlab#52

### CODE REVIEW

File: playbooks/cloud-provider-openstack-unittest-nested/run.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Deprecated (Kubernetes nested scenario)
-- hosts: all
+- name: Run unit tests with cloud-provider-openstack (nested)
+   hosts: all
```
Comment (animationzl) [inline playbooks/cloud-provider-openstack-unittest-nested/run.yaml]: A extra space ahead 'hosts' key

**Comment by liu-sheng** [comment]
yes, the check job has found that error. also not sure if need to name all the tasks, since the ansible-lint also saide "ANSIBLE0011 All tasks should be named", which I remember in openstack official repo, there also has refactor change to add names for all the tasks.

File: playbooks/cloud-provider-openstack-unittest-nested/run.yaml (hunk 1 above)
Comment (kiwik) [reply playbooks/cloud-provider-openstack-unittest-nested/run.yaml]: Good catch, thank you.

**Comment by kiwik** [comment]
@liu-sheng thank you to provider these background info, I plan to add task name in next patch.

**Comment by kiwik** [comment]
recheck

**Pull request merged**